oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexHelper.java

```diff
@@ -97,9 +104,47 @@ private static ObjectBuilder<TypeMapping> loadMappings(@NotNull TypeMapping.Buil
         if (indexDefinition.inferenceDefinition != null) {
             mapInferenceDefinition(builder, indexDefinition.inferenceDefinition);
         }
+        if (InferenceConfig.getInstance().isEnabled()) {
+            mapInferenceConfig(builder, indexDefinition, InferenceConfig.getInstance());
+        }
         return builder;
     }
 
+    private static void mapInferenceConfig(TypeMapping.Builder builder, @NotNull ElasticIndexDefinition indexDefinition, @NotNull InferenceConfig inferenceConfig) {
+        String indexName = PathUtils.getName(indexDefinition.getIndexName());
+        InferenceIndexConfig inferenceIndexConfig;
+//        try {
+        // Store the inference configuration in the index metadata so that it can be used by the inference service
```

Comment:
inference service ? Do you mean enrichment service ? Also, without context, of what an inference or an enrichment service is - the comment can be confusing.

Reply:
We have enrichconfig in inferenceConfig, hence we do have notion of enrichment service.